src/Uno.UI/UI/Xaml/Controls/DatePicker/DatePicker.md

```diff
@@ -14,19 +14,24 @@ There might be differences in the date picker on different platform since it wra
 
 #### Android
 
-Native date picker is wrapped in the flyout.
-DatePicker flyout appear centered to screen.
+- Native date picker is wrapped in the flyout.
+- DatePicker flyout appear centered to screen.
 
 #### iOS
-Native date picker is wrapped in the flyout.
-Set 'FlyoutPlacement' property to change flyout docking placement
-Default 'FlyoutPlacement' is 'Full' and will dock of the flyout at the bottom of the screen
-
+- Native date picker is wrapped in the flyout.
+- Set '**FlyoutPlacement**' property to change flyout docking placement
+- Default '**FlyoutPlacement**' is '**Full**' and will dock of the flyout at the bottom of the screen
 
 If you want to show a dimmed overlay underneath the picker, set the `DatePicker.LightDismissOverlayMode` property to `On`.
 
 If you wish to customise the overlay color, add the following to your top-level `App.Resources`:
 ```xaml
-		<SolidColorBrush x:Key="DatePickerLightDismissOverlayBackground"
-						 Color="Pink" />
+<SolidColorBrush x:Key="DatePickerLightDismissOverlayBackground"
+		Color="Pink" />
+```
+
+Since **iOS14** the native DatePicker changed the way it's presented. To keep consistency, the previous style (< iOS13) was defined as the default for this Control.
```

Comment:
Shouldn't we default to the new one and have developers add a resource if they want to keep the old one?

Reply:
@MatFillion  I did not want to take that decision. Added you as a reviewer so you could give your thoughts.

Reply:
If we default to ios13 style we will have to breaking change again in a year or two because the old style will be outdated, i would suggest using the future-proof version now as default. But both options work, you can probably confirm with jerome :)

@ajpinedam

Reply:
I'm with Mathieu, I think it makes more sense for the up-to-date iOS 14 style to be default, and the out-of-date style to be opt-in.

Reply:
@davidjohnoliver & @MatFillion  I updated the default to be the new style on iOS14